src/js/views/patients/patient/dashboard/dashboard_views.js

```diff
@@ -200,7 +200,7 @@ const FlowItemView = View.extend({
   modelEvents: {
     'change:_owner': 'render',
   },
-  behaviors: [RowBehavior],
+  behaviors: [RowBehavior, DoneBehavior],
```

Comment:
`DoneBehavior` is when the `model.state` changes to `isDone === true`. The action/flow is removed from the list with a fade-out animation.

This was never added for flow list items because it wasn't possible for that to change before. The flow state button is disabled. But now it can change via web socket notification.

Reply:
this is nice how easy that is to implement